changes/1371.removal.rst

```diff
@@ -0,0 +1 @@
+Update documentation about the iOS splash images sizes to reflect the changes in the template
```

Comment:
The change note file is the text that will be in the release notes for the next release: 
```suggestion
The size of iOS splash images have changed. iOS apps should now provide 800px, 1600px and 2400px images (previously, this as 1024px, 2048px and 3072px). This is because iOS 14 added a hard limit on the size of image resources.
```